.github/workflows/check.yml

```diff
@@ -0,0 +1,20 @@
+name: check-tools-upstream
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 * * * *'
```

Comment:
Add a comment in-line to show how frequently this is expected to run for anyone landing here who hasn't memorised cron expressions.

Reply:
okay, will update, how often do we want to run it? Once every day at 9 AM UTC?

Reply:
mid-night daily?

```
0 0 * * *
```